functions/googleGroupsManager.js

```diff
@@ -0,0 +1,24 @@
+/**
+ function that retrieves all users from the domain's gSuite
+ using Admin Directory SDK
+ * @param {OAuth2.0 instance} auth
+ * @param {gAPI instance} googleService
+ * @param {string} domain
+ * @return {Promise<any>} the found list of users or null otherwise
+ */
+exports.listUsers = async (auth, googleService, domain) => {
+  const service = googleService.admin({version: 'directory_v1', auth});
```

Comment:
`'directory_v1'` should be a constant, defined outside of this function

Reply:
Done.